docs/Makefile

```diff
@@ -0,0 +1,35 @@
+# This is a Makefile template!
+
+.SILENT:
+.DEFAULT_GOAL := help
+
+COLOR_RESET = \033[0m
+COLOR_COMMAND = \033[36m
+COLOR_YELLOW = \033[33m
+COLOR_GREEN = \033[32m
+COLOR_RED = \033[31m
+
+PROJECT := #Add your project's name here!
+
+## Installs a development environment
+install: deploy
+
+## Composes project using docker-compose
+deploy:
+	docker-compose -f deployments/docker-compose.yml build
```

Comment:
We can remove this `build` line and and a --build into the `up` line bellow. 😄

Reply:
Thanks @rafaveira3 ! I'll change it here!